templates/mia-default/settings.yaml

```diff
@@ -0,0 +1,95 @@
+
+# General settings.
+general:
+  # The CyanogenMod codename of the device.
+  cm_device_codename:
+  # Preferred CyanogenMod release type
+  cm_release_type: snapshot
+  # Preferred CyanogenMod release version
+  cm_release_version: M12
+
+
+# Mapping of the items in the definition and their place in the update archive.
+update_content:
+  - src: init.d
+    dst: system/etc/init.d
+  - src: META-INF
+    dst: META-INF
+  - src: misc
+    dst: sdcard/misc
```

Comment:
Ah-ha! OK, it's that this directory was changed, but `updater-script` wasn't adjusted to move a different file...! The confusing part was that `/sdcard` is persistent between reboots, and so the old files were still around with stale values from previous install. OK, should be able to get this merged now :)